### ISSUE TITLE
Integrating Chisel 3 blocks

### ISSUE THREAD

**Issue by rockobonaparte** [Chisel-Team/Chisel#508]
I think this is a stretch since I don't think the licensing is clear, but I otherwise hoped you could incorporate blocks from Chisel 3:

https://github.com/TheCricket/Chisel-3

It appears that project is in hiatus and it's stuck on 1.9.4. I was looking into updating to 1.10.2, but it's going to be less trivial than an updated Forge build. I figured I'd inquire before I got too deep into it.

**Comment by tterrag1098** [comment]
Such as?

**Comment by rockobonaparte** [comment]
Do you mean which blocks?

I will post back later with a more comprehensive list, but since you wrote back so soon, I wanted to give something to chew on. I have little tool at home I wrote that's telling me which blocks I'm missing out of my quest book when I move it to 1.10.2. However, I may have used more of these blocks in my tutorial dimension and haven't gotten that list yet. That being said, I know I at least am using:

- factory
- torch
- futura
- warning
- paper

It looks like you have some of these, but I do not think you have warning and torch in particular.

**Comment by rockobonaparte** [comment]
These are missing blocks that I was using in my 1.7.10 quest book when highlighting Chisel 2, which I understand to have become Chisel 3:

chisel:torch9
chisel:present', 'damage': 2
chisel:holystone
chisel:jackolantern14
chisel:fantasyblock', 'damage': 10
chisel:warningSign'

I think I have more in the tutorial dimension I created, but I am not set up to load that into 1.10.2 yet. I expect that'll also vomit out a ton of bad blocks, but hopefully the log lists each out one-by-one. I don't think I'll have that information for a day or two.

**Comment by Drullkus** [comment]
Yes, these blocks were removed for varying reasons. I could re-add them at a later date but I am extremely busy this month.

**Comment by rockobonaparte** [comment]
What a coincidence! I'm busy this month too! 🍹 

If you are planning to reincorporate them then that takes some flak off of me in doing my mod pack migration. Are you planning to just go down the Chisel 3 block list or do you still want to know specific families of blocks? It looks like I will have to actually open up the old template save and just look through everything manually to see what blocks I'm using in there. There isn't a good tool out there to automatically fetch all that. The save NBT data is going to show every chisel block in the mod because it just is translating blocks to numeric IDs. So it's useless for this.

FWIW the modpack is a tutorial pack. I have Chisel in there to make the tutorial dimension pretty, but more importantly I have a chapter on making things look pretty. I have a bunch of chisel quests in there, but I have to admit it's pretty basic since there isn't that many steps to Chisel hehe.
